src/styles/blocks/calendar.scss

```
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cell-gap;
+  max-width: 7 * ($cell-size + $cell-gap);
+
+  &__cell {
+    position: relative;
+    @include square($cell-size);
+    border: 1px solid #000;
+    background-color: #eee;
+    transition: 0.5s;
```

don't use it like that, because it's the same as transition: all 0.5s or transition-duration: 0.5s
It's too hard for the browser
Also, create variable for transition-duration
```suggestion
    transition: transform $transitionDuration, ...;
```